provider/cluster/kube/client.go

```diff
@@ -183,9 +185,25 @@ func (c *client) Deploy(ctx context.Context, lid mtypes.LeaseID, group *manifest
 
 	for svcIdx := range group.Services {
 		service := &group.Services[svcIdx]
-		if err := applyDeployment(ctx, c.kc, newDeploymentBuilder(c.log, c.settings, lid, group, service)); err != nil {
-			c.log.Error("applying deployment", "err", err, "lease", lid, "service", service.Name)
-			return err
+
+		persistent := false
+		for i := range service.Resources.Storage {
+			attrVal := service.Resources.Storage[i].Attributes.Find(StorageAttributePersistent)
+			if persistent, _ = attrVal.AsBool(); persistent {
+				break
+			}
+		}
+
+		if !persistent {
```

Comment:
Can this be arranged so it isn't inverted? Just use `if persistent` followed by `else`

Reply:
💯